### PR TITLE
[Core] Embedded Edges for BrepSurface

### PR DESCRIPTION
**Pull request by tteschemacher**
**Description**
This PR adresses the need to add additional parameter space described curves to a specific Brep Surface.

### CODE REVIEW

File: kratos/input_output/cad_json_input.h [hunk 1]
```diff
@@ -343,6 +348,21 @@ class CadJsonInput : public IO
...
     }
 
+    static void ReadAndAddEmbeddedEdges(
+            typename BrepSurfaceType::Pointer pBrepSurface,
+            const Parameters rParameters,
+            typename NurbsSurfaceType::Pointer pNurbsSurface,
+            ModelPart& rModelPart,
+            SizeType EchoLevel = 0)
+    {
+        if (rParameters.Has("embedded_edges")) {
```
Comment (rickyaristio) [inline kratos/input_output/cad_json_input.h:358]: One comment from my side. The error comes when the parameter has "embedded_edges" which is empty.

Comment (tteschemacher) [reply kratos/input_output/cad_json_input.h:358]: true, thank you very much for that hint.

**Review by rickyaristio — APPROVED** [review]

**Review by philbucher — APPROVED** [review]
ok for me :+1:

**Comment by tteschemacher** [comment]
coolio

**Pull request merged**